test/connectivity/modules/NetworkController/DockerHelper.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System;
+    using System.Net.NetworkInformation;
+    using Docker.DotNet;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class DockerHelper
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<DockerHelper>();
+        const int DockerNetworkInterfaceNameLenght = 12;
+        const int DockerNetworkInterfaceStartIndex = 0;
+
+        public static Option<string> GetDockerInterfaceName()
+        {
+            var client = new DockerClientConfiguration(new Uri(Settings.Current.DockerUri)).CreateClient();
+            var networkId = client.Networks.InspectNetworkAsync(Settings.Current.NetworkId).Result.ID;
+
+            string interfaceName = GetInterfaceNameFromNetworkId(networkId);
+
+            Log.LogInformation($"{Settings.Current.NetworkId} network has id {networkId}");
+
+            foreach (NetworkInterface item in NetworkInterface.GetAllNetworkInterfaces())
+            {
+                if (item.Name.Contains(interfaceName))
```

Comment:
Is Contains the right operation?

Reply:
Changed to equals and added br- as suffix

---
In reply to: [355819732](https://github.com/Azure/iotedge/pull/2040#discussion_r355819732) [](ancestors = 355819732)